tests/parsing/errors/expressions/object.res

```diff
@@ -0,0 +1 @@
+let obj = {"\055": "octal escape", "\123": "another octal escape"}
```

Comment:
The leading `\0` gets parsed as a `NUL`.  The contents of object.res are:
```

  Syntax error!
  tests/parsing/errors/expressions/object.res:1:13-14

  1 │ let obj = {"\055": "octal escape", "\123": "another octal escape"}
  2 │ 

  The only valid numeric escape in strict mode is '\0'


  Syntax error!
  tests/parsing/errors/expressions/object.res:1:37

  1 │ let obj = {"\055": "octal escape", "\123": "another octal escape"}
  2 │ 

  The only valid numeric escape in strict mode is '\0'

let obj =
  [%obj { <NUL>55 = {js|octal escape|js}; \123 = {js|another octal escape|js} }]% 
```
where `<NUL>` is the a char of char code 0.  Because of this character, GitHub thinks that the file is a binary file and won't show it.  Having a test case that doesn't render in GitHub is not great.  Maybe the grammar test is sufficient for testing error cases.